content/en/docs/Advanced/sys-extensions.md

```diff
@@ -0,0 +1,156 @@
+```text
+.
+└── v1.29.2+k3s1
+    └── usr
+        ├── lib
+        │   └── extension-release.d
+        │       └── extension-release.k3s-v1.29.2+k3s1
+        └── local
+            ├── bin
+            │   ├── crictl -> ./k3s
+            │   ├── ctr -> ./k3s
+            │   ├── k3s
+            │   └── kubectl -> ./k3s
+            └── lib
+                └── systemd
+                    └── system
+                        ├── k3s-agent.service
+                        └── k3s.service
+```
+
+Then you can use the `systemd-repart` tool to create the sysext image:
+```bash
+$ systemd-repart -S -s SOURCE_DIR NAME.sysext.raw --private-key=PRIVATE_KEY --certificate=CERTIFICATE       
+```
+
+{{% alert title="Warning" %}}
+Note that the extensions MUST have a `/usr/lib/extension-release.d/extension-release.NAME` file in which the NAME needs to match the sysext NAME (extension is ignored). This is an enforcement by systemd to ensure the sysext is correctly identified and some sanity checks are done with the info in that file.
+{{% /alert %}}
+
+This will generate a signed+verity sysextension that can then be used by sysext to extend the system.
+
+Some extension examples are available under https://github.com/Itxaka/sysext-examples for k3s and sbctl.
+
+
+### Verifying the system extensions
+
+
+You can use `systemd-dissect` to verify the system extension, the ID, ARCHITECTURE and the partitions that are included in the system extension.
+
+```bash
+$ sudo systemd-dissect sbctl-0.14.sysext.raw
+      Name: sbctl-0.14.sysext.raw
+      Size: 21.0M
+ Sec. Size: 512
+     Arch.: x86-64
+
+Image UUID: 351f0e17-35e5-42ff-bf09-8db65c756f7b
+ sysext R.: ID=_any
+            ARCHITECTURE=x86-64
+
+    Use As: ✗ bootable system for UEFI
+            ✗ bootable system for container
+            ✗ portable service
+            ✗ initrd
+            ✓ sysext for system
+            ✓ sysext for portable service
+            ✗ sysext for initrd
+            ✗ confext for system
+            ✗ confext for portable service
+            ✗ confext for initrd
+
+RW DESIGNATOR      PARTITION UUID                       PARTITION LABEL        FSTYPE                AR>
+ro root            4afae1e5-c73c-2f5a-acdc-3655ed91d4e0 root-x86-64            erofs                 x8>
+ro root-verity     abea5f2f-214d-4d9f-83f8-ee69ca7614ba root-x86-64-verity     DM_verity_hash        x8>
+ro root-verity-sig bdb3ee65-ed86-480c-a750-93015254f1a7 root-x86-64-verity-sig verity_hash_signature x8>
+```
+
+
+### Adding system extensions to build medium
+
+Check the `Bundling system extensions during the installable medium build` section in the [Trusted Boot Installations]({{%relref "/docs/Installation/trustedboot#bundling-system-extensions-during-the-installable-medium-build" %}}) for more information on how to add system extensions to the build medium.
```

Comment:
Since it's not code I'd go for italics or double quotes rather than backticks here

```suggestion
Check the _Bundling system extensions during the installable medium build_ section in the [Trusted Boot Installations]({{%relref "/docs/Installation/trustedboot#bundling-system-extensions-during-the-installable-medium-build" %}}) for more information on how to add system extensions to the build medium.
```